build/testing/applitools/run-msft-docs.js

```diff
@@ -0,0 +1,156 @@
+"use strict";
+
+let spawn = require('child_process').spawnSync;
+
+/**
+ * Get Branch Name using child_process and Git which runs Asynchronously
+ */
+class Config {
+    constructor() {
+        this.domainName = "https://msft-docs.fast-dna.net";
+        this.appName = "FAST-DNA";
+        this.siteName = "MSFT Doc Site";
+    }
+
+    /**
+     * Retrieves the Git Branch for the local user account
+     */
+    branchName() {
+        return spawn('git', ['rev-parse', '--abbrev-ref', 'HEAD']).output[1].toString().trim();
+    }
+}
+exports.Config = Config;
+
+/**
+ * Testing script that can be used across services  
+ */
+class Run {
+    constructor(eyes, driver, website) {
+
+        console.log("Starting tests on: %s", website);
+
+        driver.get(website);
+
+        // eyes.checkWindow("Before any action");
+        // Note: There are many ways to nagivate/find elements on a page
+        // unfortunately, not all are cross-browser compliant and so
+        // for example, we can't use xpath. As a result, we're using
+        // navigation though not necessarily most performant but effective.
+
+        // After browsers matrix is determined we can fine tune and
+        // optimize to use other selectors to see what works. If we're
+        // Testing on modern browsers this isn't as much a limitation.

```

Comment:
Can we write a TODO? This seems like something we should be tracking.

Reply:
Yanked out.  We did complete our browser matrix so this isn't needed.